_includes/configuration/editable_class.md

```diff
@@ -1,6 +1,6 @@
-### `noneditable_editable_class`
+## `editable_class`
 
-This option allows you to specify the class name that {{site.productname}} will use to determine which areas of content are editable when using the `noneditable` plugin. This would be the the same as `contenteditable=true`, forcing an element to be editable.
+This option allows you to specify the class name that {{site.productname}} will use to determine which areas of content are editable. This would be the the same as `contenteditable=true`, forcing an element to be editable.
```

Comment:
```suggestion
This option allows you to specify the class name that {{site.productname}} will use to determine which areas of content are editable. This would be the same as `contenteditable=true`, forcing an element to be editable.
```